SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -2539,16 +2540,22 @@ UpdateDeletePage (
   EFI_SIGNATURE_LIST  *CertList;
   EFI_SIGNATURE_DATA  *Cert;
   UINT32              ItemDataSize;
-  CHAR16              *GuidStr;
+  CHAR16              *CertificateInfoStr;
+  CHAR8               *CertificateInfoStr8;
   EFI_STRING_ID       GuidID;
   EFI_STRING_ID       Help;
+  UINTN CertificateInfoStrSize = 100;
+  UINTN CertificateInfoStrSizeHalf = CertificateInfoStrSize/2;

```

Comment:
I think this isn't required, if I'm wrong please add a comment why it is.

Reply:
I added CertificateInfoStrSize to not use magic numbers. CertificateInfoStrSizeHalf is here only to save up unnecesary divisions where half of CertificateInfoStrSize is needed.

Reply:
Additionally the `X509GetSubjectName` function requires the size of the buffer to be given as a `UINT8*` so we can't pass an immediate value there. We need to have the value in memory.